<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_filme_5</class>
 <widget class="QMainWindow" name="tela_filme_5">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="tela_filme_05">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1071</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 900 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QFrame" name="inicial">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>1141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(34, 31, 31);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="btn_Cine_filme">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 900 22pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>CineFilmes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Recomendacao_aleatorio_8">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Recomendação aleatório</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Inicio">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);
color: rgb(255, 0, 0);
background-color: rgb(34, 31, 31);</string>
      </property>
      <property name="text">
       <string>Inicio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Filmes_Series">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);
color: rgb(255, 0, 0);
background-color: rgb(34, 31, 31);</string>
      </property>
      <property name="text">
       <string>Filmes e Séries</string>
      </property>
     </widget>
     <widget class="QLabel" name="btn_Icon_Perfil">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../qrc/filmecinco.qrc">:/icon4/icon_perfil.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="img_Pousando_No_Amor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmecinco.qrc">:/icon1/pousando no amor.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="img_Shop_For_Killers">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>71</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmecinco.qrc">:/icon2/shop for killer.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="img_Duna_Profecia">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>81</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmecinco.qrc">:/icon 3/duna serie.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Pousando_No_Amor">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(144, 194, 255);
color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Pousando No Amor</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Shop_For_Killers">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Shop for Killer</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_2019_2020_1h_Romance">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>2019-2020 I 1h I Romance</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Duna_Profecia">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Duna: Profecia</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Ye_Jin_Son_Hyun_Bin_Ji_Hye_Seo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco:  Ye-jin Son, Hyun Bin, Ji-hye Seo</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_Pousando_No_Amor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Um acidente de parapente leva uma herdeira sul-coreana à Coreia do Norte. Lá, ela acaba conhecendo um oficial do exército, que vai ajudá-la a se esconder.&lt;/span&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Nacionalidade_Coreana">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nacionalidade: Coreana</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_2024_1h_Drama_Suspense_Acao">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2024 I 1h I Drama, Suspense</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Lee_Dong_Wook_Kim_Hye_Jun_Park_Ji_bin">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco: Lee Dong Wook, Kim Hye Jun,Park Ji-bin</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Nacionalidade_Coreana_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>360</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nacionalidade: Coreana</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_Shop_For_Killers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;A história gira em torno de Jeong Ji-an, uma jovem cuja vida aparentemente comum muda drasticamente depois que seu tio, Jeong Jin-man, morre inesperadamente.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Desde_2024_1h_Ficcao_Cientifica">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>85</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Desde: 2024 I 1h I Ficção Científica</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Emily_Watson_Olivia_Williams_Chloe_Lea">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>110</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco: Emily Watson, Olivia Williams, Chloe Lea</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Nacionalidade_EUA">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>140</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nacionalidade: EUA</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_Duna_Profecia">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>210</y>
       <width>491</width>
       <height>101</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt; Se passando 10.000 anos antes dos eventos dos filmes. A história será baseada no livro Sisterhood of Dune, escrito por Brian Herbert e Kevin J. Anderson, e segue duas irmãs Harkonnen enquanto elas combatem forças que ameaçam o futuro da humanidade e estabelecem a lendária seita que se tornará conhecida como Bene Gesserit.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_01">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>490</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>01</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_02">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>490</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>02</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_03">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>490</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>03</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_04">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>490</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>04</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_05">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>490</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>05</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../qrc/filmecinco.qrc"/>
 </resources>
 <connections/>
</ui>
